spk/sabnzbd/src/service-setup.sh

```diff
@@ -50,5 +51,7 @@ service_postupgrade ()
         if [ -n "${WATCHED_FOLDER}" ] && [ -d "${WATCHED_FOLDER}" ]; then
             set_syno_permissions "${WATCHED_FOLDER}" "${GROUP}"
         fi
+        # DSM6 -> DSM7 migration
+        sed -i -e "s|script_dir\s*=.*|script_dir = ${SYNOPKG_PKGVAR}/scripts|g" ${CFG_FILE}
```

Comment:
Since this is an upgrade, shoudln't this only trigger if `script_dir=/usr/local/sabnzbd/var/scripts`?

Reply:
Or `script_dir = /usr/local/sabnzbd/var/scripts` (so with spaces).

Reply:
Actually a good question. At first I was with you but consider that a user updates their package on DSM6 the path is changed to `/var/packages/sabnzbd/target/var` then later the update to DSM7 package gets reinstalled the new path is `/var/packages/sabnzbd/var`

So it covers both upgrade paths.

Reply:
Or in other words I know that `${SYNOPKG_PKGVAR}/scripts` is always the correct path. Setting it every time ensures it is the correct one, and I don't potentially leave out a case where it is not modified.

Reply:
But, what if the user has set a custom path to somewhere on it's NAS?

Reply:
@Safihre is it documented that this path is user changeable, if not, how can we be sure that there is a single user out there that has changed it?

Reply:
Users can easily change it from the SABnzbd interface. The example is on Windows, but on DSM it would show the `/var/.....` path from the `ini`.
![image](https://user-images.githubusercontent.com/5703454/127223492-3c1ff356-cab0-405c-a25c-23db49656563.png)